lib/eslint.js

```diff
@@ -553,11 +555,12 @@ module.exports = (function() {
 
             // If the message includes a leading line number, strip it:
             var message = ex.message.replace(/^line \d+:/i, "").trim();
+            var source = (ex.lineNumber) ? text.split("\n")[ex.lineNumber - 1].replace("\r", "") : null;
```

Comment:
Sounds good. will add a static function on `SourceCode` class which can be used to split the code into lines. Does that sound good @nzakas ?